spec/license_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'rakelib/default_plugins'
+
+describe "Plugin licenses" do
+
+  let(:common_licenses) {
+    Regexp.union([ /mit/,
+                   /apache*/,
+                   /bsd/,
+                   /ruby/])
+  }
+
+  shared_examples "a license test" do
+
+    subject(:gem_name) do |example|
+      example.metadata[:example_group][:parent_example_group][:description]
+    end
+
+    let(:spec) { Gem::Specification.find_all_by_name(gem_name)[0] }
```

Comment:
I think it fails because it uses two different methods in ruby, one is actively looking for specs ```matching_specs``` and the other is just doing ```to_spec```, but I did not more research.